package.json

```diff
@@ -4,26 +4,35 @@
   "description": "Shows Wikipedia article preview in a popup",
   "main": "dist/wikipedia-preview.production.js",
   "scripts": {
-    "test": "mocha --require @babel/register",
-    "test:watch": "mocha --require @babel/register -w",
+    "test": "mocha test/unit --require @babel/register",
+    "test:watch": "mocha test/unit --require @babel/register -w",
     "coverage": "nyc npm test",
     "build:dev": "webpack --mode development",
     "build:prod": "webpack --mode production",
     "build": "npm run build:dev && npm run build:prod",
-    "dev": "webpack-dev-server --mode development"
+    "dev": "webpack-dev-server --mode development",
+    "selenium": "wdio test/e2e/config/wdio.conf.js"
   },
   "author": "Wikimedia Foundation",
   "license": "MIT",
   "devDependencies": {
     "@babel/core": "^7.8.3",
     "@babel/preset-env": "^7.8.3",
     "@babel/register": "^7.8.3",
+    "@wdio/cli": "5.23.0",
+    "@wdio/crossbrowsertesting-service": "^5.0.0",
+    "@wdio/local-runner": "5.23.0",
+    "@wdio/mocha-framework": "5.23.0",
+    "@wdio/spec-reporter": "5.23.0",
+    "axios": "^0.19.2",
     "babel-loader": "^8.0.6",
     "bufferutil": "^4.0.1",
     "canvas": "^2.6.1",
     "css-loader": "^3.4.2",
     "eslint": "^6.8.0",
     "eslint-loader": "^3.0.3",
+    "expect": "^26.0.1",
+    "expect-webdriverio": "^1.1.5",
```

Comment:
never tried but asking here, do we use `expect` `expect-webdriverio` `@wdio/mocha-framework` here? or does it require in the current status?